scripts/util/convert.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+import csv
+import ast
+import json
+
+
+def csv_to_json(csv_path: Path, json_path: Path, sample_id: str):
+    """
+    Convert a CSV file to a JSON file using sample_id as key in the JSON
```

Comment:
If it makes it easier, you can just make the JSON an array of rows - almost exactly the output from csv.DictReader? I assume the nested dict in the csv causes an issue there?